registry/domain/factory/service_factory.py

```diff
@@ -26,7 +26,7 @@ def convert_service_db_model_to_entity_model(service):
             project_url=service.project_url,
             proto=service.proto,
             metadata_uri=service.metadata_uri,
-            assets=service.assets,
+            media=service.assets,
```

Comment:
should nt this be service.media ?

Reply:
@anandrgit my bad removed factory class itself, changes are only in the service model only